client/src/components/Overview/StyleSelector.jsx

```diff
@@ -0,0 +1,42 @@
+
+import { StyledStyleSelector } from '../styles/Overview/StyleSelector.styled';
+import Cart from './Cart';
+import { StyleGrid } from '../styles/Overview/StyleGrid.styled';
+import { StyleGridItem } from '../styles/Overview/StyleGridItem.styled';
+import { useState } from 'react';
+
+const StyleSelector = ({productStyle}) => {
+  const [selectedStyle, setSelectedStyle] = useState(productStyle.results[0]);
+
+  let handleClick = (key) => {
+    setSelectedStyle(productStyle.results[key]);
+  };
+
+  return (
+    <StyledStyleSelector>
+      <div className='product-price'>
+        <p className='last-price'>Price: <span>${selectedStyle['original_price']}</span></p>
+        {selectedStyle['sale_price'] !== null && <p className='new-price'>Sell Price: <span>$175</span></p>}
```

Comment:
Is this sell price hard coded? This will need to be dynamic

Reply:
Yeah, good catch. It should be dynamic, thank you